README.md

```diff
@@ -53,7 +54,8 @@ weather === "sunny";
 ```
 
 What are the values of these expressions?  Explain your answers.
-
+1st value is a string assigned to the variable. 2nd is a false boolean value
+because weather strictly does not equal the string sunny.
```

Comment:
it will return true because the value of weather is the string 'sunny'.

So when you do `weather ==== 'sunny'` it will return true because weather is 'sunny'